doc/sphinx/advanced_methods.rst

```diff
@@ -1847,9 +1848,12 @@ and for the determination of the density of states with respect
 to the reaction coordinate or with respect to some other collective
 variable :cite:`landsgesell17a`. Here the 1/t Wang-Landau
 algorithm :cite:`belardinelli07a` is implemented since it
-does not suffer from systematic errors. Additionally to the above
+does not suffer from systematic errors. 
+Additionally to the above
 commands for the reaction ensemble use the following commands for the
-Wang-Landau reaction ensemble. For a description of the available methods see :mod:`espressomd.reaction_ensemble`:
+Wang-Landau reaction ensemble. 
+Multiple reactions and multiple collective variables can be set.
+For a description of the available methods see :mod:`espressomd.reaction_ensemble`:
 
 .. _Constant pH simulation using the Reaction Ensemble:
```

Comment:
I find this section title rather confusing: the section title suggests that one can use the reaction ensemble to simulate a constant pH. What it describes below it the Constant-pH ensemble, implemented as a method inside the ReactionEnsemble class of Espresso. Please, here and below strictly distinguish between the `ReactionEnsemble` as a name for the respective class in Espresso, and the names of the actual thermodynamic ensembles: "reaction ensemble" a "constant-pH ensemble".

Reply:
True, I opened an issue for this #3284

Reply:
This is part of another issue, resolving it here.